components/Video.tsx

```diff
@@ -7,7 +7,7 @@ interface IVideo {
 
 export const Video: FC<IVideo> = ({ className = "", src }) => {
   return (
-    <div className="relative pb-[56.25%] h-0 overflow-hidden max-w-full">
+    <div className="relative pb-[56.25%] h-0 overflow-hidden max-w-full rounded-md">
```

Comment:
We should add this border radius to the theme, alongside the card and button border radiuses.